pkg/config/proxy.go

```diff
@@ -2,18 +2,18 @@ package config
 
 // ProxyConfig specifies the properties required to run the proxy
 type ProxyConfig struct {
-	StorageType           string `validate:"required" envconfig:"ATHENS_STORAGE_TYPE"`
-	OlympusGlobalEndpoint string `envconfig:"OLYMPUS_GLOBAL_ENDPOINT"`
-	Port                  string `validate:"required" envconfig:"PORT"`
-	FilterOff             bool   `validate:"required" envconfig:"PROXY_FILTER_OFF"`
-	BasicAuthUser         string `envconfig:"BASIC_AUTH_USER"`
-	BasicAuthPass         string `envconfig:"BASIC_AUTH_PASS"`
-	ForceSSL              bool   `envconfig:"PROXY_FORCE_SSL"`
-	ValidatorHook         string `envconfig:"ATHENS_PROXY_VALIDATOR"`
-	PathPrefix            string `envconfig:"ATHENS_PATH_PREFIX"`
-	NETRCPath             string `envconfig:"ATHENS_NETRC_PATH"`
-	GithubToken           string `envconfig:"ATHENS_GITHUB_TOKEN"`
-	HGRCPath              string `envconfig:"ATHENS_HGRC_PATH"`
+	StorageType    string `validate:"required" envconfig:"ATHENS_STORAGE_TYPE"`
+	GlobalEndpoint string `envconfig:"ATHENS_GLOBAL_ENDPOINT"`
```

Comment:
This could be confusing to some people, since we don't really have a global endpoint yet. Would be nice to add a note saying "this feature is not yet implemented"